src/widgets/Runner.ts

```diff
@@ -265,6 +272,34 @@ async function writeIframeDoc(iframe: HTMLIFrameElement, source: string, errorLi
 	});
 }
 
+// TODO: capture file and line numbers
```

Comment:
If we aren't going to do it in this PR, we should open an issue for tracking and reference it here.